code/game/atoms.dm

```diff
@@ -681,6 +685,7 @@ var/list/blood_splatter_icons = list()
 
 //the sight changes to give to the mob whose perspective is set to that atom (e.g. A mob with nightvision loses its nightvision while looking through a normal camera)
 /atom/proc/update_remote_sight(mob/living/user)
+	user.sync_lighting_plane_alpha()

```

Comment:
Why do we have to do this, out of curiosity? TG doesn't.

Reply:
That's to make sure update_remote_sight always updates the lighting plane alpha, as they usually change the user's lighting_alpha. This way you don't need to repeat it several times in each update proc, and it's not done in update_sight() due to the return on the check.